src/Utils/CameraUtils.js

```diff
@@ -252,7 +252,8 @@ class CameraRig extends THREE.Object3D {
             if (params.callback) {
                 params.callback(this);
             }
-            targetCoord.set(view.referenceCrs, this.targetWorldPosition).as(tileLayer(view).extent.crs, this.coord);
+            targetCoord.crs = view.referenceCrs;
+            targetCoord.set(this.targetWorldPosition).as(tileLayer(view).extent.crs, this.coord);
```

Comment:
I think that now, may be rename `set` to `setValues`? 
it may be necessary to remove the ambiguity, on the crs which is not modified.

Reply:
I would keep it like this, because in `Extent#set` we only set the values too, not the CRS.

Reply:
ok